src/olympia/addons/templates/addons/persona_preview.html

```diff
@@ -25,7 +25,7 @@
             {# L10n: {0} is the number of users. #}
             {{ ngettext("<strong>{0}</strong> user",
                         "<strong>{0}</strong> users",
-                        addon.persona.popularity)|f(addon.persona.popularity|numberfmt)|safe }}
+                        addon.persona.popularity)|fe(addon.persona.popularity|numberfmt)|safe }}
```

Comment:
Also, since all this formatting mess has been biting us several times in the past, it would be nice to have a single test + documentation somewhere about the best practice, maybe ? (What helpers to use, etc)